console/console-init/ui/src/graphql-module/queries/AddressSpace.ts

```diff
@@ -9,8 +9,8 @@ import { removeForbiddenChars } from "utils";
 import { generateFilterPattern } from "./query";
 
 const DELETE_ADDRESS_SPACE = gql`
-  mutation delete_as($a: ObjectMeta_v1_Input!) {
-    deleteAddressSpace(input: $a)
+  mutation delete_as($as: [ObjectMeta_v1_Input!]!) {
+    deleteAddressSpaces(input: $a)
```

Comment:
there's a typo in the variable name.
```suggestion
    deleteAddressSpaces(input: $as)
```